packages/matter-node.js/src/codec/Base38.ts

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
These codecs should move to matter.js.

Reply:
Yes will be one of the next merge steps in extra PRs as planned